style/javascript.md

```diff
@@ -567,7 +567,7 @@ Method | Use...                                | ...instead of
 bind | `fn.bind(someObj, args)` | `_.bind(fn, someObj, args)`
 bind | `(a, b) => { ... }` <sup>[1](#u1)</sup> | `_.bind(function(a, b) { ... }, this)`
 bindAll | `obj.method = obj.method.bind(someObj);` <sup>[2](#u2)</sup> | `_.bindAll(someObj, "method")`
-clone | No alternative at the moment! <sup>[3](#u3)</sup> |
```

Comment:
Looks like this was the last use of footnote 3 -- can you delete that footnote? (after confirming it really was the last.)